build_docs.py

```diff
@@ -226,9 +226,11 @@ class Language:
 )
 
 PLATEX_DEFAULT = (
```

Comment:
```suggestion
LUALATEX_DEFAULT = (
```
Should the variable name be updated too?

Reply:
Thanks, I updated in #149.